### ISSUE TITLE
Modifications to Package Installation (Ubuntu 24.04, RTX4090)

### ISSUE THREAD

**Issue by cypher004** [CAMMA-public/SurgLatentGraph#5]
The given setup had several conflicts. I finally got the following set to work:

```
conda install pytorch==2.1.2 torchvision==0.16.2 torchaudio==2.1.2 pytorch-cuda=11.8 -c pytorch -c nvidia -c pytorch -c nvidia -y &&
# conda install -c dglteam/label/th21_cu118 dgl -y &&
conda install -c dglteam/label/cu118 dgl=1.1 && 
conda install -c conda-forge torch-scatter -y &&
conda install pydantic -y &&
pip install -U openmim
mim install mmdet==3.2 &&
mim install mmengine==0.7.4 &&
conda install torchmetrics -c conda-forge -y &&
conda install scikit-learn -c conda-forge -y &&
conda install prettytable -c conda-forge -y &&
conda install imagesize -c conda-forge -y &&
conda install networkx -c conda-forge -y &&
pip install opencv-python &&
conda install yapf==0.40.1 -c conda-forge -y &&
mim install mmcv==2.1
```

**Comment by adit98** [comment]
Could you please specify some details about your machine - the setup provided in the repo generally seems to work, perhaps this is a specific case?

**Comment by cypher004** [comment]
I'm working with 

1. Ubuntu 24.04.1 LTS
2. AMD Ryzen processor
3. Nvidia 4090

1. conda 24.7.1
2. Python 3.8.19


I went through a few iterations of the setup code given as an example. It ran into two primary problems a mismatch between pytorch versions (with cuda 11.8 constrained) and a conflict with mmcv.

As a consequence I manually specified the versions based on the description and the default corresponding torch-audio, etc packages (from the PyTorch website). In addition I manually installed pydantic and mmcv
